serde/src/test/kotlin/com/amazon/ionhiveserde/objectinspectors/IonStructToMapObjectInspectorTest.kt

```diff
@@ -40,11 +41,11 @@ class IonStructToMapObjectInspectorTest {
 
     @Test
     fun getMapValueElement() {
-        val struct = makeStruct()
+        val struct = struct_for("{a: 1, b: 2, c: null}")
 
         assertEquals(ION.newInt(1), subject.getMapValueElement(struct, ION.newSymbol("a")))
         assertEquals(ION.newInt(2), subject.getMapValueElement(struct, ION.newSymbol("b")))
-        assertEquals(ION.newInt(3), subject.getMapValueElement(struct, ION.newSymbol("c")))
+        assertEquals(null, subject.getMapValueElement(struct, ION.newSymbol("c")))
```

Comment:
Does this test fail before your fix and succeed after it?

Reply:
Yeah, I fall back to the commit before this PR and only modify this unit test. It shows
<img width="939" alt="Screen Shot 2022-09-07 at 5 52 53 PM" src="https://user-images.githubusercontent.com/67451029/189009905-26f1b40a-92ec-4e1e-a81d-f02cb52b65c9.png">

Reply:
Let me also provide the before/after test results for the initial issue.